src/main/java/org/broadinstitute/consent/http/resources/InstitutionResource.java

```diff
@@ -78,16 +72,15 @@ public Response getInstitution(@Auth AuthUser authUser, @PathParam("id") Integer
   @Consumes("application/json")
   @Produces("application/json")
   @RolesAllowed(ADMIN)
-  public Response createInstitution(@Auth AuthUser authUser, String institution) {
+  public Response createInstitution(@Auth DuosUser duosUser, String institution) {
     try {
-      User user = userService.findUserByEmail(authUser.getEmail());
       Institution payload = GsonUtil.getInstance().fromJson(institution, Institution.class);
       List<Institution> conflicts = institutionService.findAllInstitutionsByName(payload.getName());
       if (!conflicts.isEmpty()) {
         throw new ConsentConflictException(
             "An institution exists with the name of '" + payload.getName() + "'");
       }
-      Institution newInstitution = institutionService.createInstitution(payload, user.getUserId());
+      Institution newInstitution = institutionService.createInstitution(payload, duosUser.getUser().getUserId());
```

Comment:
Maybe not worth doing now, but if `getUser().getUserId()` is a common pattern you could add `getUserId()` to `DuosUser`.